release.nix

```diff
@@ -1,8 +1,16 @@
-{
-  pkgs ? import ./pkgs.nix
-}:
+{ pkgs ? import ./pkgs.nix }:
 
 with pkgs;
 rec {
   application = callPackage ./default.nix {};
+  docker = dockerTools.buildImage {
+    name = application.name;
+    contents = [ application ];
+    extraCommands = ''
+      mkdir -m 1777 tmp
```

Comment:
I did this as an default template thing. Does pk or pk-agent really need the temporary directory? If not, then it's not necessary to create. That being said, some apps/subprocesses will fail if `/tmp` doesn't exist in the container. We won't be able to test this until we have a working container build first with node2nix.

Reply:
yeah I guess not, we can leave it in until we know whether it uses it. I don't think it does.